index.html

```diff
@@ -519,6 +663,27 @@ <h4>Howard DaCosta</h4>
           <p class="text-muted">UROP<br/>Gender/Novels<br>Comp/Hist</p>
         </div>
       </div>
+      <div class="col-6 col-sm-4 col-md-2">
```

Comment:
We don't list people who completed less than half a term as UROPs so Ankita should be removed.